src/main/java/com/google/alpollo/Song.java

```diff
@@ -0,0 +1,29 @@
+package com.google.alpollo;
+
+/** Represents the structure of the song. */
+public class Song {
+  private final String artist;
+  private final String name;
+  private final String album;
+
+  public Song(String artist, String name, String album) {
+    this.artist = artist;
+    this.name = name;
+    this.album = album;
+  }
+
+  /** Returns artists's name of the song. */
+  public String getArtists() {
```

Comment:
It is usually better to have field names matching their getter method. So I would rename this to getArtist, or rename the field artists